### PR TITLE
Timings

### PR DESCRIPTION
**Pull request by ly29**
Not sure to merge it yet, could do with another pass.

### CODE REVIEW

**Comment by ly29** [comment]
It has some bugs so not yet.

**Comment by ly29** [comment]
So I think this has reached a basically useful state now

File: util/bgl_callback.py [hunk 1]
```diff
@@ -77,28 +76,28 @@ def callback_disable_all():
...
+
     ng_view = space.edit_tree
     if not ng_view:
         return
 
     ng_name = space.edit_tree.name
     if not (data['tree_name'] == ng_name):
         return
-    if not isinstance(ng_view, tree.SverchokReduxTree):
+    if not ng_view.bl_idname == "SvRxTree":
```
Comment (zeffii) [inline util/bgl_callback.py:92]: : )

Comment (ly29) [reply util/bgl_callback.py:92]: It solved a circular import problem 😄

**Pull request merged**